static/src/javascripts/projects/common/views/donot-use-adblock.html

```diff
@@ -2,7 +2,7 @@
     <%=messageText%></p>
 <ul class="site-message__actions u-unstyled">
```

Comment:
What is the difference between this template and `membership-message.html` template. Can we reuse it?